### ISSUE TITLE
Update User Profiles with real corporate data from SSO integration

### ISSUE THREAD

**Issue by camelspeed** [Project-Heimdall/home#152]
Update the User Profile with real corporate data from the SSO integration.

- [ ] User Profile Avatar - Default image (eg Heimdall?) if the User is not discovered in Enterprise Github.
- [ ] User First Name, Last Name, and Email are updated with real corporate info.
- [ ] My Projects is another story and not a part of this update.

**Comment by btrayn** [comment]
We actually have a default avatar image that we can use for those who haven't uploaded a photo (see attached).
![screen-shot-2016-11-18-at-10 28 50-am](https://cloud.githubusercontent.com/assets/17861032/20435423/0d222456-ad7a-11e6-81d8-137919d7e40c.png)

**Comment by onetrickwolf** [comment]
Waiting on SSO to be deployed but after that will be working on grabbing the user icon from enterprise data then this issue will be closed.
